packages/plugin-ext-vscode/src/typings/monaco/index.d.ts

```diff
@@ -0,0 +1,19 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+/// <reference types='@theia/monaco/src/typings/monaco'/>
+
+/* expose internal APIs in @theia/monaco/src/typings/monaco */
```

Comment:
it shouldn't be there, vscode is a client of theia-monaco, if there is something  to be exposed, it should be exposed , it shouldn't use extra d.ts.

Reply:
That is what it says. Don't exposer here but do so in `@theia/monaco/src/typings/monaco`.